ginga/tests/test_dask.py

```diff
@@ -0,0 +1,97 @@
+import hashlib
+import numpy as np
+import pytest
+
+da = pytest.importorskip('dask.array')
+
+from ginga import AstroImage, trcalc
+from ginga.misc import log
+
+
+class TestDask:
+    def setup_class(self):
+        self.logger = log.get_logger("TestDask", null=True)
+
+    def _2ddata(self, shape, data_np=None):
+        if data_np is None:
+            data_np = np.asarray([min(i, j)
```

Comment:
I think this can be optimized, especially if you are using sizable `shape` values:
```python
In [30]: shape = (1000, 500)

In [31]: %timeit np.min(np.indices(shape), axis=0)
2.52 ms ± 13.7 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

In [32]: %timeit np.asarray([min(i, j) for i in range(shape[0]) for j in range(shape[1])]).reshape(shape)
92.7 ms ± 198 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
```

Reply:
@ejeschke , do you not wish to address this one? Either way is fine by me, but I just want to make sure it was not overlooked.

Reply:
Whups, ok...pushed another commit.  Have a look.